web/src/ferrostar-map.ts

```diff
@@ -25,9 +25,6 @@ export class FerrostarMap extends LitElement {
   @property()
   valhallaEndpointUrl: string = "";
 
-  @property()
-  styleUrl: string = "";
-
```

Comment:
I believe this property is still necessary for users that are not bringing their own custom MapLibre map component.